src/main/resources/application.yaml

```diff
@@ -74,6 +74,8 @@ sdt:
   purge:
     cron: ${PURGE_SCHEDULE_CRON:0 15 0 * * ?}
     commitInterval: ${PURGE_COMMIT_INTERVAL:1000}
+  requeue:
+    cron: ${REQUEUE_SCHEDULE_CRON:0 20 0 * * ?}
```

Comment:
The default should be `0 */20 * * * ?`. This means "Run every 20th minute" whereas `0 20 0 * * ?` means "Run at 00:20:00".

Reply:
done